src/simplewallet/simplewallet.cpp

```diff
@@ -4521,7 +4521,7 @@ bool simple_wallet::donate(const std::vector<std::string> &args_)
   local_args.push_back(amount_str);
   if (!payment_id_str.empty())
     local_args.push_back(payment_id_str);
-  message_writer() << tr("Donating ") << amount_str << " to The Monero Project (donate.getmonero.org/44AFFq5kSiGBoZ4NMDwYtN18obc8AemS33DBLWs3H7otXft3XjrpDtQGv7SqSsaBYBb98uNbr2VBBEt7f2wfn3RVGQBEP3A).";
+  message_writer() << tr("Donating ") << amount_str << " to The Monero Project (donate.getmonero.org or 44AFFq5kSiGBoZ4NMDwYtN18obc8AemS33DBLWs3H7otXft3XjrpDtQGv7SqSsaBYBb98uNbr2VBBEt7f2wfn3RVGQBEP3A).";
```

Comment:
This just needs the address in parenthesis if any change. It's information, not instruction. In fact, why is this PR even needed?

Reply:
Not really needed, but apparently some people view donate.getmonero.org/44AF...  as a (schemeless) URL and get confused. I think parentheses would be the clearer change too.

Reply:
My main issue is the "or" which makes no sense at this point in code. It's already done the donation and it's to the address, not the domain.